websocket/platform/sim/wb.mts

```diff
@@ -0,0 +1,34 @@
+import { WebSocketServer } from "ws";
+
+export const _startWebSocketApi = (
+  onConnect: (connectionId: string) => void,
+  onDisconnect: (connectionId: string) => void,
+  onMessage: (connectionId: string, message: string) => void) => {
+
+  const port = Math.floor(Math.random() * 1000 + 3000);
```

Comment:
what happens if the random port is already used? isn't it possible to get the server select a port automatically?